### PR TITLE
Add table component

### PR DESCRIPTION
**Pull request by Saionaro**
Screenshot 2021-04-21 at 12 02 34

### CODE REVIEW

File: components/Table/Cell.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import css from "@styled-system/css";
+import styled from "styled-components";
+import { all, StyledSystemProps } from "components/system";
+
+export const Cell = styled("td")<StyledSystemProps>(
+  css({
+    backgroundColor: "white",
```
Comment (iAdramelk) [inline components/Table/Cell.tsx:7]:
```suggestion
    bg: "white",
```

File: components/Table/HeaderCell.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { all, StyledSystemProps } from "components/system";
+
+export const HorizontalHeader = styled("th")<StyledSystemProps>(
+  css({
+    boxSizing: "content-box",
+    width: [180, 200],
+    py: 3,
+    px: [2, 3],
+    textAlign: "start",
+    backgroundColor: "lightest-gray",
```
Comment (iAdramelk) [inline components/Table/HeaderCell.tsx:12]:
```suggestion
    bg: "lightest-gray",
```

File: components/Table/Separator.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import styled from "styled-components";
+import { all, StyledSystemProps } from "components/system";
+import { Row } from "./Row";
+
+export const SeparatorCell = styled("td")<StyledSystemProps>(
+  css({
+    boxSizing: "border-box",
+    textTransform: "uppercase",
+    fontWeight: "bold",
+    backgroundColor: "lightest-gray",
```
Comment (iAdramelk) [inline components/Table/Separator.tsx:12]:
```suggestion
    bg: "lightest-gray",
```

**Review by iAdramelk — APPROVED** [review]

File: components/Table/Table.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import { all, StyledSystemProps } from "components/system";
+import { Row } from "./Row";
+
+type RowElement = ReactElement<typeof Row>;
+
+interface TableProps {
+  children: RowElement | Array<RowElement | Array<RowElement>>;
+}
+
+export function Table({ children, ...props }: TableProps & BoxProps) {
```
Comment (alex-kovoy) [inline components/Table/Table.tsx:14]:
```ts
interface TableProps extends BoxProps {
 // 
}
```
This what I would expect TableProps to be, a type that describes a Table component props. Currently it describes a component that describers only children which is not a table.

**Review by alex-kovoy — COMMENTED** [review]
I know it's been merged already but I left a comment nevertheless :)

**Pull request merged**